lib/src/cli.rs

```diff
@@ -610,6 +619,47 @@ async fn usroverlay() -> Result<()> {
         .into());
 }
 
+/// Implementation of `bootc build commit`
+/// async fn lint() -> Result<()> {
+#[context("linting")]
+fn lint() -> Result<()> {
+    if !ostree_ext::container_utils::is_ostree_container()? {
+        anyhow::bail!("Not in a ostree container, this command only verifies ostree containers.");
+    }
+
+    // check for the existance of the /var/run directory
```

Comment:
The comment is slightly misplaced I think, as someone might think it was related to opening the rootfs instead of the lint check.